src/test/container-features/configs/invalid-configs/dockerfile-with-v2-local-features-no-dev-container-folder/local-features/localFeatureB/install.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+echo "Activating feature 'localFeatureB'"
+
+echo "The provided favorite color is: $FAVORITE"
+
+tee /usr/color.sh > /dev/null \
```

Comment:
Also: See feature-template for simplified version. I suggest to copy that around for clarity.